src/workers/hibp.js

```diff
@@ -0,0 +1,161 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
How do we plan on setting up the workers in the cloud? I used to use terraform to set up the infra side of things for functions

Either way, I think we need to modify the pipeline so this code can be updated in GCP separately from the rest (dockerized -> k8)

Reply:
As discussed, let's do this part first since separating data ingestion from the user-facing services should allow one to fail without impacting the other. I'll update the ADR.